src/main/resources/application.properties.EXAMPLE

```diff
@@ -15,17 +15,23 @@ nats.truststore_path=
 nats.key_password=
 nats.store_password=
 
-# messaging configuration
-metadb.request_filter_topic=
+# new request topics
+smile.request_filter_topic=
 igo.new_request_topic=
 igo.cmo_label_generator_topic=
 
+# request and sample update topics
```

Comment:
topics:

- igo.cmo_label_update_topic=
- igo.validate_request_update_topic=
- igo.validate_sample_update_topic=
- smile.request_update_topic=
- smile.sample_update_topic=

Reply:
looks like you've accidentally entered duplicates here as well